tests/framework/installer/ceph_installer.go

```diff
@@ -235,6 +235,122 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 	return err
 }
 
+// CreateK8sRookExternalCluster creates rook external cluster via kubectl
+func (h *CephInstaller) CreateK8sRookExternalCluster(namespace, firstClusterNamespace string) error {
+
+	dataDirHostPath, err := h.initTestDir(namespace)
+	if err != nil {
+		return fmt.Errorf("failed to create test dir. %+v", err)
+	}
+	logger.Infof("Creating external cluster: namespace=%q, firstClusterNamespace=%q", namespace, firstClusterNamespace)
+
+	logger.Infof("Creating namespace %s", namespace)
+	ns := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}
+	_, err = h.k8shelper.Clientset.CoreV1().Namespaces().Create(ns)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+	}
+
+	// Skip this step since the helm chart already includes the roles and bindings
+	if !h.useHelm {
```

Comment:
We aren't ever using helm for the external cluster, right? Since we're just calling this for the multi-cluster test, no need to check this.